portfolio/src/main/webapp/script.js

```diff
@@ -11,6 +13,37 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+/**
+ * Fetches message from the server and adds it to the DOM with translation.
+ */
+function getData() {
+  console.log('Fetching data.');
+  const languageCode = document.getElementById('language').value;
+  // The fetch() function returns a Promise because the request is asynchronous.
+  const responsePromise = fetch('/list-comments?language='+languageCode).then(response => response.json())
+  .then((comments) => { 
+  const commentsListElement = document.getElementById('server-comments-container');
+  commentsListElement.innerHTML = '';
+  var i;
+  for(i = 0; i<comments.length; i++){
```

Comment:
You can define this variable inside the for loop
```suggestion
  for(let i = 0; i<comments.length; i++){
```